components/templates/ListOfSuites.tsx

```diff
@@ -72,6 +73,16 @@ export const ListOfSuites = function(props: Props) {
     setHighlightedTest(testId)
   }
 
+  function setStats() {
```

Comment:
this is a messy bit to show only  stats for those tests, that user filtered by. Since the endpoint gives all stats, that's the way to recognize which ones to show